[2022-08-13 12:28:27,483] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.create_raw_data_dir manual__2022-08-13T12:28:25.753563+00:00 [queued]>
[2022-08-13 12:28:27,497] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.create_raw_data_dir manual__2022-08-13T12:28:25.753563+00:00 [queued]>
[2022-08-13 12:28:27,498] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 12:28:27,499] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-13 12:28:27,499] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 12:28:27,516] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_raw_data_dir> on 2022-08-13 12:28:25.753563+00:00
[2022-08-13 12:28:27,522] {standard_task_runner.py:52} INFO - Started process 4746 to run task
[2022-08-13 12:28:27,526] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'create_raw_data_dir', 'manual__2022-08-13T12:28:25.753563+00:00', '--job-id', '363', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmp1v4e6o7l', '--error-file', '/tmp/tmpiosxrkx2']
[2022-08-13 12:28:27,527] {standard_task_runner.py:80} INFO - Job 363: Subtask create_raw_data_dir
[2022-08-13 12:28:27,599] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.create_raw_data_dir manual__2022-08-13T12:28:25.753563+00:00 [running]> on host 4bbb34367c41
[2022-08-13 12:28:27,688] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=create_raw_data_dir
AIRFLOW_CTX_EXECUTION_DATE=2022-08-13T12:28:25.753563+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-13T12:28:25.753563+00:00
[2022-08-13 12:28:27,690] {ETL_Raizen.py:27} INFO - Dir already exists
[2022-08-13 12:28:27,691] {python.py:173} INFO - Done. Returned value was: None
[2022-08-13 12:28:27,711] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=create_raw_data_dir, execution_date=20220813T122825, start_date=20220813T122827, end_date=20220813T122827
[2022-08-13 12:28:27,739] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-13 12:28:27,793] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
